commit-graph.c

```diff
@@ -41,6 +41,9 @@
 #define GRAPH_MIN_SIZE (GRAPH_HEADER_SIZE + 4 * GRAPH_CHUNKLOOKUP_WIDTH \
 			+ GRAPH_FANOUT_SIZE + the_hash_algo->rawsz)
 
+/* Remember to update object flag allocation in object.h */
+#define REACHABLE       (1u<<15)
```

Comment:
Is there a magic constant in `cache.h` that you can use here, to avoid clashes with other commit flags in the future?

Reply:
In `object.h`, the bitflags are laid out by which file uses which flags. This updates so that `commit-graph.c` is the only one to use `1 << 15` (outside of `builtin/show-branch.c` which uses all of them for a different reason).